test/TestProjects/ResourceClients-LowLevel/Generated/ResourceGroup.cs

```diff
@@ -0,0 +1,201 @@
+            }
+        }
+
+        /// <summary> Get a group. Method should stay in `Group` subclient. </summary>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual Response GetGroup(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceGroup.GetGroup");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetGroupRequest(context);
+                return _pipeline.ProcessMessage(message, _clientDiagnostics, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get items in group. It is defined in `Item` subclient, but must be promoted to the `Group` subclient. </summary>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual async Task<Response> GetItemsAsync(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceGroup.GetItems");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetItemsRequest(context);
+                return await _pipeline.ProcessMessageAsync(message, _clientDiagnostics, context).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get items in group. It is defined in `Item` subclient, but must be promoted to the `Group` subclient. </summary>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual Response GetItems(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceGroup.GetItems");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetItemsRequest(context);
+                return _pipeline.ProcessMessage(message, _clientDiagnostics, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Initializes a new instance of Resource. </summary>
+        /// <param name="itemId"> The String to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="itemId"/> is null. </exception>
+        public virtual Resource GetResourceClient(string itemId)
```

Comment:
Should not have a `Client` suffix per: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-subclient-factory-methods-suffix

Reply:
Fixed